### PR TITLE
[DMS-322] Convert Debezium __deleted flagged record to tombstone

### PR DESCRIPTION
**Pull request by bradbanister**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,21 @@ transforms=RenameDmsTopicToOpenSearchIndex
 transforms.RenameDmsTopicToOpenSearchIndex.type=org.edfi.kafka.connect.transforms.RenameDmsTopicToOpenSearchIndex
 ```
 
+### `DebeziumDeletedToTombstone`
+
+This transformation checks for a Debezium _deleted=true flag. If found, it turns it into a tombstone
+
+- `org.edfi.kafka.connect.transforms.DebeziumDeletedToTombstone`
+  - works on values.
```
Comment (stephenfuqua) [inline README.md]: I'm not sure what this implies.

Comment (bradbanister) [reply README.md]: Me neither, will remove. Copy and paste 🤣

File: kafka/ed-fi-kafka-connect-transforms/src/main/java/org/edfi/kafka/connect/transforms/DebeziumDeletedToTombstone.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+        System.out.println("isDeleted is " + isDeleted);
+
+        return record.newRecord(
+                record.topic(),
+                record.kafkaPartition(),
+                record.keySchema(),
+                record.key(),
+                isDeleted ? null : record.valueSchema(),
+                isDeleted ? null : record.value(),
```
Comment (bradbanister) [inline kafka/ed-fi-kafka-connect-transforms/src/main/java/org/edfi/kafka/connect/transforms/DebeziumDeletedToTombstone.java:63]: Nulling out record value makes it a tombstone

File: kafka/ed-fi-kafka-connect-transforms/config/checkstyle/checkstyle.xml [hunk 3]
```diff
@@ -33,12 +33,6 @@
     <property name="localeLanguage" value="en"/>
     <property name="fileExtensions" value="java, properties, xml"/>
 
-    <!-- See http://checkstyle.sourceforge.net/config_header.html#RegexpHeader -->
-    <module name="RegexpHeader">
-        <property name="fileExtensions" value="java"/>
-        <property name="headerFile" value="${config_loc}/java.header" default="checkstyle/java.header"/>
-    </module>
-
```
Comment (bradbanister) [inline kafka/ed-fi-kafka-connect-transforms/config/checkstyle/checkstyle.xml:41]: Files have different license headers now

**Review by stephenfuqua — APPROVED** [review]

**Pull request merged**